ray/operator/base/kustomization.yaml

```diff
@@ -1,9 +1,28 @@
----
-kind: Kustomization
-apiVersion: kustomize.config.k8s.io/v1beta1
-resources:
-  - ray-operator-serviceaccount.yaml
-  - ray-operator-role.yaml
-  - ray-operator-rolebinding.yaml
-  - ray-operator-deployment.yaml
-  - ray-custom-resources.yaml
+# Adds namespace to all resources.
+namespace: ray-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+#namePrefix: ray-operator-
+
+# Labels to add to all resources and selectors.
+#commonLabels:
+#  someName: someValue
+
+bases:
+- ../crd
+- ../rbac
+- ../manager
+- ../prometheus
+- namespace.yaml
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+#- ../prometheus
+
+images:
+- name: kuberay/operator
+  newName: kuberay/operator
+  newTag: nightly
```

Comment:
@MichaelClifford Is there any concern with referencing a `nightly` tag breaking any deployments?

Reply:
Good point @LaVLaS This is what the KubeRay repo suggests, but we can pin this down to a recent version.